app/index.js

```diff
@@ -168,14 +169,16 @@ Generator.prototype.gitIsTheShit = function() {
 			if (err) me.logger.error(err);
 
 			me.logger.verbose('Git init complete');
-			git.add('.', function(err) {
-				if (err) me.logger.error(err);
-			}).commit('Initial Commit', function(err, d) {
-				if (err) me.logger.error(err);
+			done();
+			//git.add('.', function(err) {
+				//if (err) me.logger.error(err);
+
+			//}).commit('Initial WP Commit', function(err, d) {
+				//if (err) me.logger.error(err);
```

Comment:
Since we are doing 2.0, changing the behavior is just fine.  I must have missed the commit later down.  You should leave the new behavior, I like it being all done at once later.